src/components/summary.vue

```diff
@@ -20,11 +20,11 @@
             class="fr-sidemenu__item"
           >
             <router-link
-              v-if="!disabledLink(chapter, index)"
+              v-if="!disabledLink(chapter)"
```

Comment:
On peut directement mettre cette méthode non ?
```suggestion
              v-if="!isChapterPending(chapter)"
```

Reply:
L'avantage que je vois a créé une method disabledLink, c'est que si on rajoute de la logique plus compliquée à l'intérieur on force le future dev à se faire dans `disabledLink` ce qui évite de potentiellement complexifier le `v-if`. 

Par ailleurs je trouve que c'est pas évident en regardant le template qu'on "disable" un lien.